CHANGELOG.md

```diff
@@ -35,6 +39,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 https://tls.mbed.org/tech-updates/releases/mbedtls-2.16.5-and-2.7.14-released)
 release notes for the set of issues addressed.
 
+### Deprecated
```

Comment:
Thinking about it a little more, I think calling it out in deprecated is wise since this is what developers will look at in order to plan for the next release.